versioned_docs/version-2.x/magic-button/x-ray.mdx

```diff
@@ -37,6 +37,16 @@ When enabled, content composing the page are outlined with a user interface
 allowing the contributor to know the source of it and providing a quick access
 to the corresponding administration interface.
 
+:::tip
+
+When using Front-Commerce with Magento 1 as backend you can set the env variable
+`FRONT_COMMERCE_MAGENTO_TYPE` to `magento1` or `openmage` to specify to Mangeto
+1 backend type (by default `openmage`).
+
+It will only change color of border content to match the color of your backend
```

Comment:
```suggestion
When using Front-Commerce with a Magento 1 backend, you can set the `FRONT_COMMERCE_MAGENTO_TYPE` environment variable to `magento1` or `openmage` which will specify Magento 1 as your backend type (by default `openmage`).

It will change the X-Ray colors to match the Magento 1 default color.
```

Reply:
It might be good to have this environment variable more explicit as it might be confusing in the future as to what it is achieving, for example `FRONT_COMMERCE_STOREFRONT_MAGENTO_VERSION` or `FRONT_COMMERCE_XRAY_MAGENTO_VERSION`